app/src/main/java/com/bignerdranch/android/criminalintent/CrimeFragment.kt

```diff
@@ -64,6 +76,9 @@ class CrimeFragment: Fragment(), DatePickerFragment.Callbacks {
             Observer { crime ->
                 crime?.let {
                     this.crime = crime
+                    photoFile = crimeDetailViewModel.getPhotoFile(crime)
+                    photoUri = FileProvider.getUriForFile(requireActivity(),
+                    "com.bignerdranch.android.criminalintent.fileprovider", photoFile)
```

Comment:
`"com.bignerdranch.android.criminalintent.fileprovider"` seems to be a constant here, therefore it would make sense to put it as a `private const val` in companion object

Reply:
My mistake, you already pointed it in previous review